setup.py

```diff
@@ -32,9 +32,13 @@
     packages=['edx_rest_framework_extensions'],
     install_requires=[
         'django>=1.8.9,<2.0',
-        'djangorestframework>=3.2.3,<4.0.0',
+        'djangorestframework==3.6.3',
         'djangorestframework-jwt>=1.7.2,<2.0.0',
         'python-dateutil>=2.0',
         'requests>=2.7.0,<3.0.0',
+        'six==1.11.0',
     ],
+    dependency_links=[
+        'git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3'
```

Comment:
This pins us to the edx fork of DRF just as edx-platform has. Prior it was pulling in latest DRF which no longer supports 1.9